index.js

```diff
@@ -74,13 +74,16 @@ export const createConfig = async({
 
   if (Platform.OS === 'ios') {
     scopes = scopes.join(' ');
+    const defaultiOSTimeout = 15;
```

Comment:
Does the timeout need to be different on iOS and Android? Or can this be shared between them, for consistency?

Reply:
Android handled timeout internally in native bridge. But I refactored JS code according to your suggestion.
Thanks!